bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/HeavyNeutrino_trilepton_V-0p01_e_NLO/HeavyNeutrino_trilepton_mNgtmWltmZ_V-0p01_e_NLO/HeavyNeutrino_trilepton_V-0p01_e_NLO_customizecards.dat

```diff
@@ -0,0 +1,11 @@
+set param_card mass 9900012 HNLMASS
+set param_card numixing 1 1.000000E-02
+set param_card numixing 2 0.000000e+00
+set param_card numixing 3 0.000000e+00
+set param_card numixing 4 0.000000e+00
+set param_card numixing 5 0.000000e+00
+set param_card numixing 6 0.000000e+00
+set param_card numixing 7 0.000000e+00
+set param_card numixing 8 0.000000e+00
+set param_card numixing 9 0.000000e+00
+compute_widths 9900012
```

Comment:
please use "set param_card decay 9900012 Auto" instead
compute_widths is buggy sometimes

Reply:
We have used compute_widths in all the gridpacks we have already submitted for official production... It is probably more consistent to keep the same settings for the VBF samples.

Reply:
why should this be more consistent? the auto setting will work for sure. if compute_width will cause problems, we can resubmit the old ones. please fix. the gen_checking script will veto the compute_width logic. you cannot submit such requests at present.